pkg/secretstore/datastore_keys.go

```diff
@@ -0,0 +1,126 @@
+package secretstore
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-datastore"
+	"github.com/libp2p/go-libp2p/core/crypto"
+
+	"berty.tech/weshnet/pkg/errcode"
+)
+
+const (
+	// dsNamespaceChainKeyForDeviceOnGroup is a namespace stores the current
+	// state of a device chain key for a given group.
+	// It contains the secret used to derive the next value of the chain key
+	// and used to generate a message key for the message at `counter` value,
+	// then put in the dsNamespacePrecomputedMessageKeys namespace.
+	dsNamespaceChainKeyForDeviceOnGroup = "chainKeyForDeviceOnGroup"
+
+	// dsNamespacePrecomputedMessageKeys is a namespace storing precomputed
+	// message keys for a given group, device and message counter.
+	// As the chain key stored has already been derived, these message keys
+	// need to be computed beforehand.
+	// The corresponding message can then be decrypted via a quick lookup.
+	dsNamespacePrecomputedMessageKeys = "precomputedMessageKeys"
+
+	// dsNamespaceMessageKeyForCIDs is a namespace containing the message key
+	// for a given CID once the corresponding message has been decrypted.
+	dsNamespaceMessageKeyForCIDs = "messageKeyForCIDs"
+
+	// dsNamespaceOutOfStoreGroupHint is a namespace where HMAC value are
+	// associated to a group public key.
+	// It is used when receiving an out-of-store message (e.g. a push
+	// notification) to identify the group on which the message belongs, which
+	// can then be decrypted.
+	dsNamespaceOutOfStoreGroupHint = "outOfStoreGroupHint"
+
+	// dsNamespaceOutOfStoreGroupHintCounters is a namespace storing first and
+	// last counter values for generated group hints inside the
+	// dsNamespaceOutOfStoreGroupHint namespace
+	dsNamespaceOutOfStoreGroupHintCounters = "outOfStoreGroupHintCounters"
+
+	// dsNamespaceGroupDatastore was used to gather data related to groups into
+	// a single sub-datastore, it has been kept for retro-compatibility
+	dsNamespaceGroupDatastore = "account_groups_datastore"
```

Comment:
If it's no longer used, removing the old namespaces should be better to keep the code simple.
Can you put in the PR description that this PR breaks the retro compact?